src/spec/index.spec.ts

```diff
@@ -157,7 +157,7 @@ describe('actions', () => {
   });
 
   it('selects sub state', () => {
-    const state: {
+    const featureState: {
```

Comment:
Can you change this in `globalState`? The feature state is the `myFeature` property.